<launch>
  <arg name="param_file_path" default="$(find-pkg-share gnss_area_filter)/config/gnss_area_filter.param.yaml" />

  <arg name="input_lanelet2_map_bin_topic" default="/map/vector_map" />
  <arg name="input_pose_with_covariance_topic" default="/localization/util/gnss_distortion_corrector/pose_with_covariance" />

  <node pkg="gnss_area_filter" exec="gnss_area_filter" name="gnss_area_filter">
    <remap from="input_lanelet2_map_bin_topic" to="$(var input_lanelet2_map_bin_topic)" />
    <remap from="input_pose_with_covariance_topic" to="$(var input_pose_with_covariance_topic)" />

    <param from="$(var param_file_path)" />
  </node>
</launch>
